### PR TITLE
update modules used in cheyenne cmake testing

### PR DESCRIPTION
**Pull request by jedwards4b**
Update the external software levels of cmake tests on cheyenne.

### CODE REVIEW

**Comment by jedwards4b** [comment]
@edhartnett the failing test was added in a previous PR and seems to be passing, this PR should not have affected it at all - any idea why it fails here?

File: tests/cunit/CMakeLists.txt [hunk 1]
```diff
@@ -233,6 +235,10 @@ else ()
     EXECUTABLE ${CMAKE_CURRENT_BINARY_DIR}/test_darray
     NUMPROCS ${AT_LEAST_FOUR_TASKS}
     TIMEOUT ${DEFAULT_TEST_TIMEOUT})
+  add_mpi_test(test_darray_uneven
```
Comment (edhartnett) [inline tests/cunit/CMakeLists.txt]: Well here is the issue. You added this test to the cmake build and it is failing.

File: tests/cunit/CMakeLists.txt [hunk 2]
```diff
@@ -68,6 +68,8 @@ if (NOT PIO_USE_MPISERIAL)
   target_link_libraries (test_pioc_fill pioc)
   add_executable (test_darray EXCLUDE_FROM_ALL test_darray.c test_common.c)
   target_link_libraries (test_darray pioc)
+  add_executable (test_darray_uneven EXCLUDE_FROM_ALL test_darray_uneven.c test_common.c)
```
Comment (edhartnett) [inline tests/cunit/CMakeLists.txt]: This test does not exist in the code. If you created it, try git add test_darray_uneven.c. If not, then delete all references to it in cmake.

Comment (jedwards4b) [reply tests/cunit/CMakeLists.txt]: oops - I thought that I had added it. I will correct - thanks.

File: tests/cunit/CMakeLists.txt (hunk 1 above)
Comment (jedwards4b) [reply tests/cunit/CMakeLists.txt]: It works locally - how do I see details of the travis run?

**Pull request closed without merging**